Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.21-alpine AS builder
+FROM golang:1.23-alpine AS builder
```

Comment:
`1.24` is available; did you specifically want 1.23, or is the newer one fine too?

Reply:
1.24 is fine. I bumped to 1.24.